### PR TITLE
Added an apply page for project managers

### PR DESCRIPTION
**Pull request by liammoyn**
Still need quotes from Chloe, Tonina, and others for testimonials. Any suggestions on the writing is very appreciated

Screen Shot 2021-04-29 at 9 41 52 AM

### CODE REVIEW

File: src/pages/apply/project-manager/ApplyProjectManager.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    paddingRight: '0',
+    maxWidth: '16em',
+  },
+  application: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  text: {
+    //fontFamily: '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
```
Comment (florisdobber) [inline src/pages/apply/project-manager/ApplyProjectManager.tsx:35]: Why is this commented out? Just remove if not necessary?

File: src/pages/apply/project-manager/ApplyProjectManager.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    Organized and forward thinking
+                  </li>
+                  <li>Passionate about helping the Boston community</li>
+                  <li>
+                    A strong communicator who facilitates collaboration amongst
+                    the product team
+                  </li>
+                  <li>
+                    Interested in being a part of the entire product life cycle:
+                    software inception to completion
```
Comment (florisdobber) [inline src/pages/apply/project-manager/ApplyProjectManager.tsx]: Not sure about "software inception". Just inception could work, or maybe something like "from proposal to implementation to completion". Not a great example, but something like that? "software inception" just feels like it's not a thing

File: src/pages/apply/project-manager/ApplyProjectManager.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+                  </li>
+                </ul>
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```
Comment (florisdobber) [inline src/pages/apply/project-manager/ApplyProjectManager.tsx:148]: I know this is just copy-pasted from the other application pages but maybe we want to update this for next semester.

**Review by florisdobber — APPROVED** [review]
Looks good, just don't forget to add the testimonials before merging

**Review by rymaju — APPROVED** [review]
lgtm

File: src/pages/apply/project-manager/ApplyProjectManager.tsx (hunk 1 above)
Comment (liammoyn) [reply src/pages/apply/project-manager/ApplyProjectManager.tsx:35]: Not sure, this is copied from the software developer page, @rymaju ?

File: src/pages/apply/project-manager/ApplyProjectManager.tsx (hunk 2 above)
Comment (liammoyn) [reply src/pages/apply/project-manager/ApplyProjectManager.tsx]: I made it sounds nicer

File: src/pages/apply/project-manager/ApplyProjectManager.tsx (hunk 3 above)
Comment (liammoyn) [reply src/pages/apply/project-manager/ApplyProjectManager.tsx:148]: Yeah we can edit all the apply pages at once with some PR

**Pull request merged**